modules/parser/src/synchronizers/xviz-stream-buffer.js

```diff
@@ -335,6 +348,31 @@ export default class XVIZStreamBuffer {
   }
   /* eslint-enable complexity, no-unused-expressions */
 
+  _mergePersistentSlice({streams}) {
+    const oldStreams = (this.persistent && this.persistent.streams) || {};
+    const newStreams = Object.assign({}, oldStreams);
+
+    for (const streamName in streams) {
+      assert(
```

Comment:
Does this get removed in production code? Still not sure how if this is a soft error or hard error.

Reply:
Technically the collision doesn't break anything, just the non-persistent updates will be ignored. We can move it into the builder?